impl/protobuf-ts/runner.ts

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env -S npx tsx
+
+import {
+  ConformanceRequest,
+  ConformanceResponse,
+  FailureSet,
+  TestCategory,
+  WireFormat,
+} from "./gen/conformance/conformance.js";
+import { TestAllTypesProto3 } from "./gen/google/protobuf/test_messages_proto3.js";
+import { TestAllTypesProto2 } from "./gen/google/protobuf/test_messages_proto2.js";
+import { readSync, writeSync } from "fs";
+import { Any } from "./gen/google/protobuf/any.js";
+import { Duration } from "./gen/google/protobuf/duration.js";
+import { FieldMask } from "./gen/google/protobuf/field_mask.js";
+import { Int32Value } from "./gen/google/protobuf/wrappers.js";
+import { Struct } from "./gen/google/protobuf/struct.js";
+import { Timestamp } from "./gen/google/protobuf/timestamp.js";
+import { Value } from "./gen/google/protobuf/struct.js";
+
+const registry: Record<string, any> = {
+  "google.protobuf.Any": Any,
+  "google.protobuf.Duration": Duration,
+  "google.protobuf.FieldMask": FieldMask,
+  "google.protobuf.Int32Value": Int32Value,
```

Comment:
We can do better with protobuf-ts, see [registry here](https://github.com/timostamm/protobuf-ts/blob/8c167ae34395f72f6c95385c3760a9d7d90775aa/packages/test-generated/conformance_ts.ts#L15) and lookup [by name here](https://github.com/timostamm/protobuf-ts/blob/8c167ae34395f72f6c95385c3760a9d7d90775aa/packages/test-generated/conformance_ts.ts#L34).

Reply:
My bad.  I should've just copied the conformance runner from that repo in the first place.  Updated.